acceptance/tests/base/dsl/helpers/web_helpers/web_helpers_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'helpers/test_helper'
+require 'webrick'
+require 'webrick/https'
+
+test_name 'dsl::helpers::web_helpers #link_exists?' do
+
+  http_cmd = "ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 80, :DocumentRoot => Dir.pwd).start' > /tmp/mylogfile 2>&1 &"
+  host = hosts[0]
```

Comment:
Is there any expectation that `hosts[0]` is a particular role, or just any host will work?